torch/_C/_VariableFunctions.pyi.in

```diff
@@ -3,25 +3,33 @@
 # mypy: allow-untyped-defs
 
 import builtins
+from types import EllipsisType
```

Comment:
`ellipsis` in `builtins` is deprecated. Import `EllipsisType = type(Ellipsis)` explicitly.